ui/src/main/resources/org/pentaho/di/ui/core/default.properties

```diff
@@ -8,6 +8,7 @@ BackgroundColorB=255
 BackgroundColorG=255
 BackgroundColorR=255
 DefaultPreviewSize=1000
+DisableBrowserEnvironmentCheck=false
```

Comment:
Are you sure about this? All other values seem to be `Y`/`N` instead of `true`/`false`.